logspout/image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM flynn/busybox
+FROM alpine
```

Comment:
I feel like this should be a separate PR, or at the very least, a separate commit. It's not really related to this PR, is it?